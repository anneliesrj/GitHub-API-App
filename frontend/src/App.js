import React from 'react';
import UserSearch from './components/UserSearch'; // Import UserSearch component

function App() {
  return (
    <div className="App">
      <main>
        {/* Render the UserSearch component */}
        <UserSearch /> 
      </main>
    </div>
  );
}

export default App;
