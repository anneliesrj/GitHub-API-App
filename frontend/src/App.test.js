import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

test('render test', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
