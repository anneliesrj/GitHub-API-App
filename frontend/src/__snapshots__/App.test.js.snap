// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render test 1`] = `
<div
  className="App"
>
  <main>
    <div
      className="user-info-container"
    >
      <header
        className="App-header"
      >
        <img
          alt="GitHub logo"
          src="https://cdn.pixabay.com/photo/2022/01/30/13/33/github-6980894_1280.png"
        />
        <h1>
          Github User Search
        </h1>
      </header>
      <div
        className="search"
      >
        <input
          onChange={[Function]}
          placeholder="Enter GitHub username"
          type="text"
          value=""
        />
        <button
          onClick={[Function]}
        >
          Search
        </button>
      </div>
    </div>
     
  </main>
</div>
`;
